supplements/xanathars-guide-to-everything/eldritch-invocations.xml

```diff
@@ -214,10 +226,16 @@
 		<description>
 			<p class="flavor">Prerequisite: 7th level</p>
 			<p>You can cast freedom of movement once on yourself without expending a spell slot. You regain the ability to do so when you finish a long rest.</p>
+			<div class="reference">
+				<div element="ID_PHB_SPELL_FREEDOM_OF_MOVEMENT" />
+			</div>
 		</description>
 		<sheet usage="1/Long Rest">
 			<description>You can cast freedom of movement once on yourself without expending a spell slot.</description>
 		</sheet>
+		<rules>
+			<grant type="Spell" id="ID_PHB_SPELL_FREEDOM_OF_MOVEMENT" />
+		</rules>
```

Comment:
this is probably the only arguable Invocation that shouldn't give Freedom of Movement, because it can be argued you don't know the spell since you can't cast it normally, and even if RAI is that it should count as known, i would prefer to not grant it for the sake of those DMs who don't agree with that interpretation (since those who would agree could easily add the spell via Equipment section).